src/Google.Events.Protobuf/Cloud/Firestore/V1/Data.g.cs

```diff
@@ -69,8 +69,10 @@ public static partial class DataReflection {
             "KAsyNi5nb29nbGUuZXZlbnRzLmNsb3VkLmZpcmVzdG9yZS52MS5NYXBWYWx1",
             "ZS5GaWVsZHNFbnRyeRpWCgtGaWVsZHNFbnRyeRILCgNrZXkYASABKAkSNgoF",
             "dmFsdWUYAiABKAsyJy5nb29nbGUuZXZlbnRzLmNsb3VkLmZpcmVzdG9yZS52",
-            "MS5WYWx1ZToCOAFCLKoCKUdvb2dsZS5FdmVudHMuUHJvdG9idWYuQ2xvdWQu",
-            "RmlyZXN0b3JlLlYxYgZwcm90bzM="));
+            "MS5WYWx1ZToCOAFCdqoCKUdvb2dsZS5FdmVudHMuUHJvdG9idWYuQ2xvdWQu",
```

Comment:
There are no extension attributes generated for Firestore and a few others, like CloudBuild and Storage. Just flagging in case that's not expected.

Reply:
> There are no extension attributes generated for Firestore and a few others, like CloudBuild and Storage. Just flagging in case that's not expected.

Just checked those three, and events.proto doesn't declare any. (I suspect they may be added later, as I'm pretty sure they should have them.) That won't be breaking, so I'm not too worried :)